public_dropin_notebook_environments/python311_notebook_base/Dockerfile

```diff
@@ -54,10 +54,10 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # Add any package that will be installed on system level here:
 RUN echo "[Python$PYTHON_VERSION] GIT_COMMIT: $GIT_COMMIT" \
-    && microdnf update \
-    && microdnf install -y python$PYTHON_VERSION-$PYTHON_EXACT_VERSION python$PYTHON_VERSION-devel-$PYTHON_EXACT_VERSION \
-  gcc-8.5.0 gcc-c++-8.5.0 glibc-devel-2.28 libffi-devel-3.1 graphviz-2.40.1 python$PYTHON_VERSION-pip \
-  openblas-0.3.15 python$PYTHON_VERSION-scipy shadow-utils passwd git openssh-server tar gzip unzip zip wget \
+    && microdnf update -y \
+    && microdnf install -y python$PYTHON_VERSION-$PYTHON_EXACT_VERSION python3-devel-3.9.21 \
```

Comment:
This is odd to me but works - I needed python 3.9 devel in order for uwsgi to build correctly.

But once I built image and uploaded to DR and used in a Codespace the githelper finally worked and the python version was _actually_ 3.11

<img width="1198" alt="Screenshot 2025-04-10 at 12 47 55 PM" src="https://github.com/user-attachments/assets/cfe7294c-e7d9-4065-815f-6b7cabc2c81d" />